_posts/2021-08-17-cultural-competency-in-archives-investigating-archival-description-for-av-materials.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Learning Together: Cultural Competency in Archives: Investigating Archival Description for Audiovisual Materials
+date:2021-08-17T11:30:00
```

Comment:
hi! I just noticed there should be a space here after `date: `

Reply:
thanks for the catch!